platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -31,6 +31,9 @@
 #define _ARM_CRYPTO_MAX_CIPHER_KEY_LENGTH      64
 #define _ARM_CRYPTO_MAX_AUTH_KEY_LENGTH        64
 #define _ARM_CRYPTO_MAX_IV_LENGTH              16
+/* Data must be a multiple of the block size which is 16 bytes. */
```

Comment:
I think this is slightly misleading since it is not the data size but the buffer size that has to be a multiple of 16. Also, the comment does not mention where the requirement comes from.

I would remove the comment and add these lines after the constant definitions.

/*
 * Data buffer size must be a multiple of 16, because the ARM crypto
 * library will write full 16 byte blocks even if the last data block
 * is not a full block.
 */
ODP_STATIC_ASSERT(ARM_CRYPTO_MAX_DATA_LENGTH % 16 == 0);

Reply:
Sounds good, I will make the change. Thanks!